pkg/operator/ceph/disruption/machinedisruption/add.go

```
@@ -32,8 +32,12 @@ import (
 // https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg
 func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 	mgrScheme := mgr.GetScheme()
-	healthchecking.AddToScheme(mgrScheme)
-	cephv1.AddToScheme(mgrScheme)
+	if err := healthchecking.AddToScheme(mgrScheme); err != nil {
+		logger.Errorf("failed to add to healthchecking scheme. %v", err)
```

Let's return these errors